AB#312 Vaasa service area

tuomas-lindell opened the pull request:
## Proposed Changes

  - Updated Vaasa service area

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in an issue in Azure Boards
  - Merge the pull request

tuomas-lindell committed:
AB#312 Vaasa service area

vesameskanen reviewed (changes requested):
- Vaasa service area is still a rectangle. Just update minLat ...maxLon values. Rectangle is much faster geometry to limit geocoding search and requires less data.
- I can see some formatting changes which violate our es-lint rules. For some strange reason, many configs have eslint disabled in the first line of config. Please remove that comment line and fix all es-lint errors while you update the service area  in each config. Note that you can apply yarn es-lint --fix to address most lint errors.

tuomas-lindell commented:
- changed service area back to rectangle
- Restored old file formatting, removed es-lint comment and did run yarn es-lint --fix, now result after running es-lint formatting is in committed file (formatting looks to me same as other config-files)

vesameskanen reviewed (changes requested):
I think minLat and maxLat have reversed values.
